config/element-api.php

```diff
@@ -217,11 +217,148 @@ function getLegacyPage($locale)
     ];
 }
 
+function getBasicEntryData($entry) {
+    $basicData = [
+        'id' => $entry->id,
+        'path' => $entry->uri,
+        'url' => $entry->url,
+        'title' => $entry->title
+    ];
+
+    // @TODO this is duplicated below
+    if ($entry->mainHeading) {
+        $basicData['mainHeading'] = $entry->mainHeading;
+    }
+
+    if ($entry->photo) {
+        $photos = [];
+        foreach ($entry->photo->all() as $photo) {
+            $photos[] = $photo->url;
+        }
+        if ($photos) {
+            $basicData['photo'] = $photos[0];
+        }
+    }
+
+    return $basicData;
+}
+
+function getRelatedEntries($entry, $relationType) {
+    $relatedEntries = [];
+    if ($relationType == 'children') {
+        $relatedSearch = $entry->getChildren()->find();
```

Comment:
`->find()` seems to be deprecated, logs mention using `->all()` instead.

Reply:
👍